src/supertux/console.cpp

```diff
@@ -384,7 +385,7 @@ Console::autocomplete()
   // depending on number of hits, show matches or autocomplete
   if (cmds.empty())
   {
-    m_buffer.addLines("No known command starts with \"" + prefix + "\"");
+    m_buffer.addLines(_("No known command starts with \"") + prefix + _("\""));
```

Comment:
I realize it's used there; however I don't think we need translations in that place. If we do, all other console output (including warnings/errors) should be translated as well.